source/octod/core.d

```diff
@@ -305,12 +307,14 @@ struct HTTPConnection
             }
         );
 
-        this.handleResponseStatus(response);
+        if (auto location = this.handleResponseStatus(response))
+            return this.patch(location, json);
```

Comment:
This changes seems to belong to another commit (Support HTTP 302 responses (redirects)).